### ISSUE TITLE
missing Cabral tree

### ISSUE THREAD

**Issue by ahhurlbert** [sELDIG/SimulationStudy#3]
Currently missing ca_83.tre from @julianoscabral simulation output

**Comment by julianoscabral** [comment]
The tree will be keeping missing though, because I tried already 3 times and the computer crashes trying to create the phylogeny for that particular replicate. :-( Not sure this experiment is worth counting as environmental filtering anyway though. In principle it is, but it was not a model parameter in the strict sense that was changed, just the simulation landscape (the low filtering being the landscape with more habitat types). One could argue that changing the environment is changing a key parameter, and that changing the landscape indeed mimics how environmental filtering happens in nature though. I have another experiment that could be classified as high (spatial) environmental filtering, but it changes further assumptions (like the temporal availability of the habitats). This brings the whole full factorial design issue. These simulations may take long, like a week per run, if I simulate oceanic islands with realistically paced environmental dynamics. I will set up some experiments that are easier or faster, still I need a lot more time. For the moment the analyses must consider the output from the experiments that are already published and that can be somehow categorized. For those, dis and tim (as 3 snap shots within a replicate) are the most straightforward. Note that the medium dis scenario was done in two different ways: either by changing the dispersal ability or by increasing geographical distance. That is why I have double the amount of replicates for medium (n=40). I guess we need to explain the experiments in the google doc.

**Comment by julianoscabral** [comment]
Ok, that brings the issue that one can simulate env in different ways even with the same model. So I have 3 different ways to simulate high environmental filtering from the temporally dynamic, spatially heterogenous landscapes: 1) faster environmental dynamics, but same environmental heterogeneity, 2) no environmental dynamics with less spatial heterogeneity, 3) same temporal dynamic, but without spatial hererogeneity. It would be interesting to classify all these as env ='high', compared to the reference, but the patterns may be very very different and collapsing them into one category may cancel out the signals. However, I do like the idea of comparing different ways to simulate env filtering, as much as how I do with dis. I guess the idea was to do this across models, but within models is also possible.

**Comment by ahhurlbert** [comment]
Yes, I totally agree and think that everyone should feel free to run experiments within submodels like this. For example, I mentioned to @florianhartig that he could run a speciation rate experiment within one set of simulations using point mutation speciation, and a different set of simulations with fission speciation. You could run 'env' experiments in these 3 different ways. I think we can examine those cases as though they were separate models, at least initially.
